tests/foreman/ui/test_settings.py

```diff
@@ -353,9 +353,44 @@ def test_positive_update_email_delivery_method_sendmail():
     :CaseImportance: Critical
 
     :CaseLevel: Acceptance
-
-    :CaseAutomation: notautomated
     """
+    mail_config_default_param = {
+        "delivery_method": "",
+        "email_reply_address": "",
+        "email_subject_prefix": "",
+        "sendmail_arguments": "",
+        "sendmail_location": "",
+        "send_welcome_email": "",
+    }
+    for content in mail_config_default_param:
+        mail_config_default_param[content] = [
+            ele.value
+            for ele in entities.Setting().search(query={'search': 'name={}'.format(content)})
+        ][0]
+
+    mail_config_new_params = {
+        "delivery_method": "Sendmail",
+        "email_reply_address": gen_email(),
+        "email_subject_prefix": [gen_string('alpha')],
+        "sendmail_location": "/usr/sbin/sendmail",
+        "send_welcome_email": "Yes",
+    }
+
+    with session:
+        try:
+            for mail_content in mail_config_new_params:
+                session.settings.update(
+                    "{}".format(mail_content), "{}".format(mail_config_new_params[mail_content])
+                )
+            contents = session.settings.contents()
+            contents.email_button.click()
+            assert contents.notification_locator.text == "Email was sent successfully"
+        finally:
+            for mail_content in mail_config_default_param:
+                setting_cleanup(
+                    setting_name=mail_content,
+                    setting_value=mail_config_default_param[mail_content],
+                )
```

Comment:
Please use `items()` to iterate over key and value at once:
```suggestion
            for key, value in mail_config_default_param.items():
                setting_cleanup(setting_name=key, setting_value=value)
```

Reply:
Fixed.